module-services/service-evtmgr/AppSettingsNotify.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include "Service/BusProxy.hpp"
+#include <application-settings/ApplicationSettings.hpp>
+
+namespace evm::api
+{
+    void notifySettingsBluetoothAudio(sys::BusProxy &bus, std::shared_ptr<audio::Event> event)
```

Comment:
Why is it moved to different file? Isn't it out of scope for this task?

Reply:
This comment is twice: https://github.com/mudita/MuditaOS/pull/3648#discussion_r827718886